a3p-integration/proposals/a:upgrade-next/probeZcfBundleCap.test.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @jessie.js/safe-await-separator */
```

Comment:
I think this isn't doing anything. also the best remedy is `await null` instead of suppression